test/parallel/test-buffer-fill.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
+const errors = require('../../lib/internal/errors');// Flags: --expose-internals
```

Comment:
(Also, it means that in order to run the test from the command line, you need to either use `tools/test.py` or else pass the `--expose-internals` flag to `node`. But lots of tests are like that already, so that should be fine.)